### DISCUSSION TITLE
Worth jumping on the beta?

### DISCUSSION THREAD

**Discussion by v-marinkov** [monicahq/monica#6718] [General]
I have 45 contacts in my Monica 4.0 instance and I am wondering whether I should manually migrate to the beta before my database grows too much and locks me in the old version. Is the beta stable enough for casual use? Will future updates fully support the beta's database? Is there a Docker image?

Many thanks

**Comment by meichthys** [comment]
I don't think there's a rush since an export/import features are planned. AFAIK you will be able to migrate your Monica data to Chandler.

**Comment by bbrendon** [comment]
I looked at chandler a few weeks ago and I remember the take-away I got was that it looks clean but has very few features to the point where it wasn't usable.

**Reply by v-marinkov** [reply]
Yeah, I got the same impression. In fact, I couldn't figure out how to record a conversation I had with someone. I'll stick to the old version for now then.

**Reply by DJFriar** [reply]
Calls are there, but they are slightly hidden: Click on a contact, then go to Information; Calls is near the bottom. However, other Activities do seem to be missing.
